src/index.js

```diff
@@ -153,6 +153,35 @@ const definedProps = [
   'handleWrapperClass', 'children', 'onResizeStart', 'onResize', 'onResizeStop', 'handleComponent',
 ];
 
+const body = document.body;
+let resizeOverlay = null;
+
+const addResizeOverlay = (cursor: string) => {
+  resizeOverlay = document.createElement('div');
+  resizeOverlay.style.cssText = `
+    height: 100%;
+    width: 100%;
+    background-color: rgba(0,0,0,0);
+    cursor: ${cursor || 'auto'};
+    opacity: 0;
+    position: fixed;
+    z-index: 9999;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;`;
+  if (body) {
+    body.appendChild(resizeOverlay);
+  }
+};
```

Comment:
I think that it is better to do as follows. what do you think about it?

```
<div
        ref={(c: React.ElementRef<'div'> | null) => { this.resizable = c; }}
        style={{
          position: 'relative',
          ...userSelect,
          ...this.props.style,
          ...this.sizeStyle,
          maxWidth: this.props.maxWidth,
          maxHeight: this.props.maxHeight,
          minWidth: this.props.minWidth,
          minHeight: this.props.minHeight,
          boxSizing: 'border-box',
        }}
        className={this.props.className}
        {...this.extendsProps}
      >
        <div
           style={{
              ...some overlay styles...
              display: this.state.isResizing ? 'block' : 'none',
            }} 
        />
        {this.props.children}
        {this.renderResizer()}
      </div>
```

Reply:
Interesting :) This seems like a better approach.

I dont like rendering stuff that gets display none though - what about changing it to this:
```
{this.state.isResizing  &&
        <div
           style={{
              ...some overlay styles...
            }} 
        />}
```

to avoid rendering the resize-div when it is not needed. WDYT?

Reply:
Looks Good :)